src/components/Catalog.vue

```diff
@@ -100,9 +100,14 @@
                 striped
               >
                 <template slot="cell(link)" slot-scope="data">
-                  <router-link :to="data.item.to">{{
+                  <router-link v-if="!data.item.loading" :to="data.item.to">{{
                     data.item.title
                   }}</router-link>
+              
+                  <b-button v-if="data.item.loading" variant="secondary" disabled>
```

Comment:
I don't know if the original project maintainer would like this extra addition, I would minimize the features addition and either open another PR for this or not add it

Reply:
This was implemented in order to avoid other error related with rendering the stac items table, but yes, it makes total sense. This one and the other one with behaviour change.